spec/features/bulkrax_export_spec.rb

```diff
@@ -0,0 +1,126 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Bulkrax export', clean: true, perform_enqueued: true do
```

Comment:
Could you replace all of the ' for " ? It would take me ages to make all the suggestions :D